tests/jms-tests/src/test/java/org/apache/activemq/artemis/jms/tests/tools/container/InVMInitialContextFactory.java

```diff
@@ -31,11 +31,7 @@
 
    // Static --------------------------------------------------------
 
-   private static Map<Integer, Context> initialContexts;
-
-   static {
-      InVMInitialContextFactory.reset();
-   }
+   private static final Map<Integer, Context> initialContexts = new HashMap<>();
 
```

Comment:
This was not final because it was resettable.

Reply:
Yes, but nobody was actually resetting it, in the codebase.

Reply:
Fair enough.